sdk/src/Core/Amazon.Runtime/_bcl/ClientConfig.cs

```diff
@@ -165,7 +165,7 @@ public int MaxIdleTime
         /// <summary>
         /// Gets and sets the connection limit set on the ServicePoint for the WebRequest.
         /// Default value is 50 connections unless ServicePointManager.DefaultConnectionLimit is set in 
-        /// which case ServicePointManager.DefaultConnectionLimit will be used as the default.
+        /// which case ServicePointManager.DefaultConnectionLimit will be used as the default. This property is only available in .NET Framework: 4.5 and newer, 3.5. 
```

Comment:
![image](https://github.com/user-attachments/assets/8b2965a5-d40f-425a-a49f-15870ea36929)
I think the comment is misleading since .net standard and .netcore are newer than .net framework 4.5, probably `.NET Framework 4.5 and earlier` make more sense.

Also can we wrap this addition in `<para>` tag to make it stand out in the docs?

Reply:
sure i can update to add `<para>` tag. As for the wording, i based it on the very bottom of this page https://docs.aws.amazon.com/sdkfornet/v3/apidocs/items/S3/TS3Config.html and wanted to make it consistent.

Reply:
Can you actually just say `.NET Framework`. That way the wording works for either V3 and V4 which have different .NET Framework targets.

Reply:
Yea I like this suggestion better. I will update to just say `.NET Framework`. and ill share what it looks like with `<para>` tag too,

Reply:
Updated to just say `.NET Framework.` also spoke with muhammad offline and he mentioned `<para>` was only suggestion and not needed, so I've excluded it for now.